docs/installation/linux/debian.md

```diff
@@ -21,6 +21,12 @@ Docker is supported on the following versions of Debian:
  >**Note**: If you previously installed Docker using `APT`, make sure you update
  your `APT` sources to the new `APT` repository.
 
+This page instructs you to install using Docker-managed release packages and
+installation mechanisms. Using these packages ensures you get the latest release
+of Docker. If you wish to install using Debian-managed packages, consult your
+Debian documentation. If you are comfortable with package managers, and are familiar
+with Docker's prerequisites, feel free to skip to step 3 in the [Install](#install-docker) section
```

Comment:
I think the original issue was that we don't mention the `curl | sh` method on the Ubuntu and Debian pages; only skipping the prerequisites section doesn't solve that.

If we want to be consistent, I think we should

- Add a section for the `curl | sh` "quick install" method (possibly mentioning its limitations)
- Add a link from that section to the "manual installation" method for those comfortable with package managers to do the manual steps

Also see my comment on the linked issue https://github.com/docker/docker/issues/19235#issuecomment-171010218